tests/acceptance/16_cf-serverd/serial/simple_copy_from_deny_localhost.cf

```diff
@@ -16,8 +16,9 @@ bundle agent test
       # 127.0.0.1/::1 to localhost, the reverse lookup fails and so
       # 'deny => { "localhost" };' and 'deny_hostnames => { "localhost" };'
       # are not enough to deny access for this test.
-      "test_skip_unsupported" string => "(ubuntu_20|ubuntu_22)",
-        meta => { "ENT-2480", "ENT-7362" };
+      # Test fails on OpenSUSE 15, pending investigation.
```

Comment:
maybe adjust the existing comment to include OpenSUSE 15 instead of taking on the end? e.g.

```Ubuntu 20, 22 and OpenSUSE 15 don't seem to know how to resolve```

Reply:
The thing is that I'm not entirely sure _why_ the test fails on OpenSUSE 15 - it doesn't use systemd-resolved, as far as I was able to check, so it might be something different.

That's why I created ticket to investigate and added OpenSUSE as a separate comment